pages/apis/rest_api/analytics/flaky_tests.md

```diff
@@ -0,0 +1,46 @@
+# Flaky tests API
+
+The Flaky test API endpoint provides information about tests detected as flaky in a test suite.
+
+{:toc}
+
+## List all flaky tests
+
+Returns a [paginated list](<%= paginated_resource_docs_url %>) of the flaky tests detected in a test suite.
+
+```bash
+curl "https://api.buildkite.com/v2/analytics/organizations/{org.slug}/suites/{suite.slug}/flaky-tests"
+```
+Optional [query string parameters](/docs/api#query-string-parameters):
+<table class="fixed-width">
+  <tbody>
+    <tr>
+      <th>
+        <code>test_id</code>
+      </th>
+      <td>
+        <span>Filters the results for a specific test.</span>
+        <p class="Docs__api-param-eg"><em>Example:</em> <code>?test_id=my_test_id</code></p>
+      </td>
+    </tr>
+  </tbody>
+</table>
+
```

Comment:
There is currently no optional query parameters to filter the results.

moot point - it seems the pattern in other pages is that the 'optional query string parameters' section comes after the response example. But if we intend to remove this section, this doesn't matter.

Reply:
Oh weird, this optional endpoint was mentioned in another PR. Have removed.